src/slab.py

```diff
@@ -0,0 +1,54 @@
+"""Slab data stucture that is used to represent Order book."""
```

Comment:
Let's put this in the layouts folder that I created for account flags.

Reply:
Done